src/osip-monitoring-controller/src/main/java/edu/kit/pse/osip/monitoring/controller/MonitoringController.java

```diff
@@ -49,13 +73,31 @@ public MonitoringController() {
         currentSettings = new ClientSettingsWrapper(new File(settingsLocation, "monitoring.conf"));
         currentView = new MonitoringViewFacade();
         currentSettingsView = new SettingsViewFacade(currentSettings);
+        
+        for (TankSelector selector: TankSelector.valuesWithoutMix()) {
+            TankContainer cont = new TankContainer();
+            tanks.add(cont);
+            cont.tank = currentProductionSite.getUpperTank(selector);
```

Comment:
Kannst du currentProductionSite in prodSite oder productionSite umbenennen? Das current macht irgendwie keinen Sinn, weil es doch sowieso nur eine ProductionSite gibt.